### PR TITLE
feat: update collection from excel

### PR DESCRIPTION
**Pull request by theus77**
add an update collection field option to the DocumentUpdateCommand

### CODE REVIEW

**Review by Davidmattei — COMMENTED** [review]
Nice, but why is the collection-field a param of the command? Just add it in the config?

File: src/Client/Document/Update/DocumentUpdater.php [hunk 1]
```diff
@@ -109,4 +100,65 @@ private function getRawDataFromRow(array $row): array
...
+                $ouuid = $this->getOuuidFromRow($row);
+                $rawData = $this->getRawDataFromRow($row);
+                if ($this->io->isVerbose()) {
+                    $this->io->note(\sprintf('Update document %s', $ouuid));
+                    $this->io->note(Json::encode($rawData, true));
+                }
+                if (!$this->dryRun) {
+                    $dataApi->save($ouuid, $rawData, DataInterface::MODE_UPDATE, false);
+                }
+                exit;
```
Comment (Davidmattei) [inline src/Client/Document/Update/DocumentUpdater.php]: ???
```suggestion
```

**Review by Davidmattei — CHANGES_REQUESTED** [review]

File: src/Client/Document/Update/DocumentUpdater.php (hunk 1 above)
Comment (theus77) [reply src/Client/Document/Update/DocumentUpdater.php]: Oups....

Comment (theus77) [reply src/Client/Document/Update/DocumentUpdater.php]: Solved

**Review by Davidmattei — APPROVED** [review]

**Pull request merged**